web/src/components/post.astro

```diff
@@ -0,0 +1,26 @@
+---
+interface Prop {
+	frontmatter: {
+		title: string
+		author: string
+	}
+	url: string
+}
+const { frontmatter, url }: Prop = Astro.props as Prop
+const { title, author } = frontmatter
+---
+
+<article class="card">
+	<div class="relative my-8 border-2 border-lime-500">
+		<div class="ml-2 flex justify-center items-center">
+			<h3>
+				<a href={url}>{title}</a>
+			</h3>
+		</div>
+		<div class="flex justify-center items-center m-8">
+			by {author}
+		</div>
+		<!-- TODO: <p>{description}</p> -->
+	</div>
+	<break></break>
```

Comment:
Not needed if you remove `class=relative` above, which itself is no longer needed since you removed the "read" button.